### ISSUE TITLE
Rule: Curly braces should not be used in PHTML files 

### ISSUE THREAD

**Issue by renttek** [extdn/extdn-phpcs#16]
Curly braces should not be used in PHTML files.

Bad:
```php
<?php
if ($block->getFooList()) {
    foreach ($block->getFooList() as $foo) {
?>
        <!-- Template stuff -->
<?php
    }
}
?>
```

Better (Proposal 1):
```php
<?php
if ($block->getFooList()):
    foreach ($block->getFooList() as $foo):
?>
        <!-- Template stuff -->
<?php
    endforeach;
endif;
?>
```

Better (Proposal 2):
```php
<?php if ($block->getFooList()) ?>
    <?php foreach ($block->getFooList() as $foo): ?>
        <!-- Template stuff -->
    <?php endforeach; ?>
<?php endif; ?>
```

(Source: https://twitter.com/jissereitsma/status/990301162090975232)

**Comment by jissereitsma** [comment]
Thanks @renttek . You mention that "curly braces should not be used in PHTML templates". Could you elaborate a bit on why this is the case? I understand that this could be a best practice, but we first need to summarize why this actually *needs* to be a best practice.

**Comment by navarr** [comment]
I also prefer curly braces in PHTML when (and only when) the start and end is in the same PHP block, for example, if statements and variable assignment at the top of the template file.

**Comment by schmengler** [comment]
Closed in favor of new issue #41
